rust/src/pgsql/pgsql.rs

```diff
@@ -706,19 +762,33 @@ pub extern "C" fn rs_pgsql_state_get_tx_count(state: *mut std::os::raw::c_void)
 }
 
 #[no_mangle]
-pub extern "C" fn rs_pgsql_tx_get_state(tx: *mut std::os::raw::c_void) -> PgsqlTransactionState {
+pub extern "C" fn rs_pgsql_tx_get_req_state(tx: *mut std::os::raw::c_void) -> PgsqlTxReqProgress {
```

Comment:
This function is pure rust, no extern C...

Reply:
good catch, sir 🙇🏽